### PR TITLE
ACR Admin module Initial version

### PR DESCRIPTION
**Pull request by larisaborodina**
Changed for Azure stack ACR Admin cmdlets auto generation

### CODE REVIEW

File: src/Azs.ContainerRegistry.Admin/readme.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+require:
+  - $(this-folder)/../readme.azurestack.md
+  - $(repo)/specification/azsadmin/resource-manager/containerregistry/readme.azsautogen.md
+
+metadata:
+  description: 'Microsoft AzureStack PowerShell: ContainerRegistry Admin cmdlets'
+
+### PSD1 metadata changes
+subject-prefix: ''
+module-version: 1.0.0
```
Comment (bganapa) [inline src/Azs.ContainerRegistry.Admin/readme.md:58]: pls change it to 0.1.0 for the initial release

File: src/Azs.ContainerRegistry.Admin/readme.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+      subject: Quota
+    set:
+      subject-prefix: ContainerRegistry
+
+    # Rename Set-AzsContainerRegistryQuota to Update-AzsContainerRegistryQuota
+  - where:
+      verb: Set
+      subject: ContainerRegistryQuota
+    set:
+      verb: Update
```
Comment (bganapa) [inline src/Azs.ContainerRegistry.Admin/readme.md:83]: verb Update typically implies Patch call, Set- is preferred

File: src/Azs.ContainerRegistry.Admin/readme.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+	  
+# Add release notes
+  - from: Azs.ContainerRegistry.Admin.nuspec
+    where: $
+    transform: $ = $.replace('<releaseNotes></releaseNotes>', '<releaseNotes>AzureStack Hub Admin module generated with https://github.com/Azure/autorest.powershell.</releaseNotes>');
+
+# Add Az.Accounts/Az.Resources as dependencies
+  - from: Azs.ContainerRegistry.Admin.nuspec
+    where: $
+    transform: $ = $.replace('<dependency id=\"Az.Accounts\" version=\"1.6.0\" />', '<dependency id="Az.Accounts" version="2.2.4" />\n      <dependency id="Az.Resources" version="[1.10.0]" />');
```
Comment (bganapa) [inline src/Azs.ContainerRegistry.Admin/readme.md:107]: 2.2.8 instead of 2.2.4

Comment (bganapa) [inline src/Azs.ContainerRegistry.Admin/readme.md:107]: 0.11.0 instead of 1.10.0

File: src/Azs.ContainerRegistry.Admin/readme.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+# PSD1 changes for RequiredModules
+  - from: source-file-csharp
+    where: $
+    transform: $ = $.replace('sb.AppendLine\(\$@\"\{Indent\}RequiredAssemblies = \'\{\"./bin/Azs.ContainerRegistry.Admin.private.dll\"\}\'\"\);', 'sb.AppendLine\(\$@\"\{Indent\}RequiredAssemblies = \'\{\"./bin/Azs.ContainerRegistry.Admin.private.dll\"\}\'\"\);\n      sb.AppendLine\(\$@\"\{Indent\}RequiredModules = @\(@\{\{ModuleName = \'Az.Accounts\'; ModuleVersion = \'2.2.4\'; \}\}, @\{\{ModuleName = \'Az.Resources\'; RequiredVersion = \'1.10.0\'; \}\}\)\"\);');
```
Comment (bganapa) [inline src/Azs.ContainerRegistry.Admin/readme.md:112]: similar version updates here

File: src/readme.azurestack.md [hunk 5]
```diff
@@ -4,8 +4,9 @@
 ``` yaml
 azure: true
 powershell: true
-branch: stackadmin2
-repo: https://github.com/Azure/azure-rest-api-specs/tree/$(branch)
+branch: release/acrchanges
```
Comment (bganapa) [inline src/readme.azurestack.md:7]: Need to revert

**Review by bganapa — CHANGES_REQUESTED** [review]
Need Pester tests

**Comment by bganapa** [comment]
Please change target branch to dev instead of master, we just recently changed the default branch to dev

File: src/Azs.ContainerRegistry.Admin/readme.md (hunk 2 above)
Comment (larisaborodina) [reply src/Azs.ContainerRegistry.Admin/readme.md:83]: We discussed with @rjaini about this renaming and agreed with this change.
@rjaini are we still ok with Update or follow @bganapa recommendation?

File: src/Azs.ContainerRegistry.Admin/readme.md (hunk 1 above)
Comment (larisaborodina) [reply src/Azs.ContainerRegistry.Admin/readme.md:58]: Done

File: src/readme.azurestack.md (hunk 5 above)
Comment (larisaborodina) [reply src/readme.azurestack.md:7]: Done

File: src/Azs.ContainerRegistry.Admin/readme.md (hunk 3 above)
Comment (larisaborodina) [reply src/Azs.ContainerRegistry.Admin/readme.md:107]: Done

Comment (larisaborodina) [reply src/Azs.ContainerRegistry.Admin/readme.md:107]: Done

File: src/Azs.ContainerRegistry.Admin/readme.md (hunk 4 above)
Comment (larisaborodina) [reply src/Azs.ContainerRegistry.Admin/readme.md:112]: Done

File: src/Azs.ContainerRegistry.Admin/readme.md (hunk 2 above)
Comment (rjaini) [reply src/Azs.ContainerRegistry.Admin/readme.md:83]: @larisaborodina: Initially we thought to keep like this to be in consistent with update commands of container registry. But I remember we discussed in standup to follow Azure Stack hub guidelines as that is what customer is more familiar. So based on that we should move to Set rather update as all AZS command follow the same convention.

Comment (larisaborodina) [reply src/Azs.ContainerRegistry.Admin/readme.md:83]: Done

**Review by bganapa — CHANGES_REQUESTED** [review]
Also needs docs/ examples with output

**Comment by bganapa** [comment]
@larisaborodina pls let us know if you need any inputs

**Comment by larisaborodina** [comment]
It should be merge in dev first

**Pull request closed without merging**